adserver/api/serializers.py

```diff
@@ -82,6 +85,14 @@ def validate_keywords(self, keywords):
 
         return keywords
 
+    def validate_url(self, url):
+        validator = URLValidator()
+        try:
+            validator(url)  # Throws ValidationError on invalid
+            return url
+        except ValidationError:
+            return None
```

Comment:
I think lets keep this logic, but log here to understand how often this is happening? Once we understand if its common, we can figure out what to do.